types/key/control_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package key
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestControlKey(t *testing.T) {
+	serialized := `{"PrivateKey":[36,132,249,6,73,141,249,49,9,96,49,60,240,217,253,57,3,69,248,64,178,62,121,73,121,88,115,218,130,145,68,254]}`
```

Comment:
hah, what an encoding!

/me checks that you didn't actually submit our actual prod private key to source 😛

Reply:
I didn't, don't panic :) Generated a test key with NewControl(), serialized it, manually verified the serialization matched the preexisting control plane serialization, then hardcoded the test key.